vip_hci/leastsq/leastsq.py

```diff
@@ -54,17 +54,19 @@ def xloci(cube, angle_list, fwhm=4, metric='manhattan', dist_threshold=50,
         central circular area is discarded.
     asize : int, optional
         The size of the annuli, in pixels.
-    n_segments : int or list of ints, optional
+    n_segments : int or list of int or 'auto', optional
         The number of segments for each annulus. When a single integer is given
-        it is used for all annuli.
+        it is used for all annuli. When set to 'auto', the number of segments is
+        automatically determined for every annulus, based on the annulus width.
     nproc : None or int, optional
         Number of processes for parallel computing. If None the number of
         processes will be set to (cpu_count()/2). By default the algorithm works
         in single-process mode.
-    solver : {'lstsq', 'nnls'}, str optional
+    solver : {'lstsq', 'nnls', 'lsq'}, str optional
```

Comment:
My short test with the least squares imposing bounds on the variables was not conclusive. As it is right now, lines 230-232, it's equivalent to the non negative least squares. If you'd like to investigate what benefits we could obtain with the scipy.optimize.lsq_linear function, please go ahead! Otherwise I'd prefer to avoid having 'lsq' (scipy.optimize.lsq_linear) here